### PR TITLE
feat: add custom service Healthchecks

### LINKED ISSUE #474: Add Healthchecks custom service
Hi :wave:, 
I would like to see a custom service for [Healthchecks](https://healthchecks.io/) added to Homer. It is a really helpful application for keeping track of jobs on a schedule and is available in a selfhosted variant. 

Find attached to this issue a pull request adding the service.

The API endpoint is `http://app.domain/api/v1/checks/` and the JSON response looks like the following:

```json
{
  "checks": [
    {
      "name": "name",
      "slug": "slug",
      "tags": "tags",
      "desc": "description",
      "grace": 86400,
      "n_pings": 456,
      "status": "up",
      "last_ping": "2022-06-19T13:15:24+00:00",
      "next_ping": "2022-06-25T09:00:00+02:00",
      "manual_resume": false,
      "methods": "",
      "last_duration": 152,
      "unique_key": "38jdd83ejdjd83jd8jd",
      "schedule": "0 09 * * 0,6",
      "tz": "Europe/XXXXXX"
    },
    [...]
  ]
}
```

### PR DESCRIPTION
**Pull request by Tooa**
## Description

This pull request adds [Healthchecks](https://healthchecks.io/) as a custom service to Homer. I based the style on the Portainer service for consistency. It will look like the following:

homer_healthchecks

Fixes #474

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] I've read & comply with the [contributing guidelines](https://github.com/bastienwirtz/homer/blob/main/CONTRIBUTING.md)
- [X] I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers. 
- [X] I have made corresponding changes to the documentation (README.md).
- [X] I've checked my modifications for any breaking changes, especially in the `config.yml` file

### CODE REVIEW

File: docs/customservices.md [hunk 1]
```diff
@@ -223,3 +224,16 @@ endpoint pointing to Tautulli!
...
   apikey: "MY-SUPER-SECRET-API-KEY"
 ```
+
+## Healthchecks
+
+This service displays information about the configured status checks from the Healthchecks application.
+Two lines are needed in the config.yml :
+
+```yaml
+  type: "Healhchecks"
```
Comment (cuu508) [inline docs/customservices.md:234]: Typo: Healhchecks

File: docs/customservices.md [hunk 2]
```diff
@@ -223,3 +224,16 @@ endpoint pointing to Tautulli!
...
+  apikey: "01234deb70424befb1f4ef6a23456789"
+```
+
+The url must be the root url of the Healhchecks application.
```
Comment (cuu508) [inline docs/customservices.md:238]: Typo again

File: src/components/services/Healhchecks.vue [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    </template>
+  </Generic>
+</template>
+
+<script>
+import service from "@/mixins/service.js";
+import Generic from "./Generic.vue";
+
+export default {
+  name: "Healhchecks",
```
Comment (cuu508) [inline src/components/services/Healhchecks.vue:24]: Typo: Healhchecks

File: docs/customservices.md (hunk 1 above)
Comment (Tooa) [reply docs/customservices.md]: Good catch! Fixed.

**Review by bastienwirtz — APPROVED** [review]
HI @Tooa, 
Thanks for your contribution. Everything looks good, I'm merging it :+1:

**Pull request merged**